react/features/base/config/configType.ts

```diff
@@ -489,9 +489,9 @@ export interface IConfig {
     prejoinPageEnabled?: boolean;
     raisedHands?: {
         disableLowerHandByModerator?: boolean;
-        disableLowerHandNotification?: boolean;
```

Comment:
This serves the same purpose as 'disableRemoveRaisedHandOnFocusNotification'. Since I think the new name is clearer, I’ve updated it accordingly. Please let me know any advice.